shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -188,12 +188,6 @@ public static void ensureInitializationComplete(@NonNull Context applicationCont
             return;
         }
         try {
-            // There are resources to extract. For example, the AOT blobs from the `assets` directory.
-            // `sResourceExtractor` is `null` if there isn't any AOT blob to extract.
-            if (sResourceExtractor != null) {
-                sResourceExtractor.waitForCompletion();
```

Comment:
We still need this `sResourceExtractor.waitForCompletion()` line, no?

Reply:
Done.